examples/experimental/dagster-airlift/dagster_airlift/migration_state.py

```diff
@@ -15,9 +15,16 @@ class DagMigrationState(NamedTuple):
 class AirflowMigrationState(NamedTuple):
     dags: Dict[str, DagMigrationState]
 
-    def get_migration_state_for_task(self, dag_id: str, task_id: str) -> bool:
+    def get_migration_state_for_task(self, dag_id: str, task_id: str) -> Optional[bool]:
+        if dag_id not in self.dags:
+            return None
+        if task_id not in self.dags[dag_id].tasks:
+            return None
         return self.dags[dag_id].tasks[task_id].migrated
 
+    def dag_is_being_migrated(self, dag_id: str) -> bool:
+        return self.get_migration_dict_for_dag(dag_id) is not None
+
```

Comment:
`dag_is_being_migrated` is a bit confusing. I would say something like `dag_has_migration_state` as that doesn't presume intent. (e.g. you can imagine having a `DagMigrationState` for a dag you never intended to migrate.